resources/intent/ultimaker_s3/um_s3_aa0.4_um-abs_0.2mm_engineering.inst.cfg

```
@@ -0,0 +1,28 @@
+[general]
+definition = ultimaker_s3
+name = Accurate
+version = 4
+
+[metadata]
+intent_category = engineering
+material = ultimaker_abs
+quality_type = draft
+setting_version = 22
+type = intent
+variant = AA 0.4
+
+[values]
```

Need to enable the acceleration since it is now disabled by default
```suggestion
[values]
_plugin__curaenginegradualflow__0_1_0__gradual_flow_enabled = True
```